src/Uno.UI/UI/Xaml/UIElementGeneratedProxy.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Uno.UI.Xaml;
+
+public static class UIElementGeneratedProxy
```

Comment:
@MartinZikmund Do we expect this to be used by consumers, or only generators?
If only generators, we should add `[EditorBrowsable(EditorBrowsableState.Never)]`

Reply:
Indeed, good idea